Tasks/GithubRelease/README.md

```diff
@@ -0,0 +1,74 @@
+
```

Comment:
This is incorrect content in the md file. Clear the content and work with the PM to get the right content

Reply:
removed the content for now and added todo